deploy/packages/external-libraries.yaml

```diff
@@ -28,7 +28,7 @@ packages:
       - cudnn@7.3.0
       # - freeimage@3.18.0
       - glew
-      - gmsh+metis~mpi+oce+openmp+shared@4.4.1
+      - gmsh+metis+mpi+oce+openmp+shared@4.4.1
```

Comment:
Once we depend on `mpi`, could we move this into `parallel-libraries`?

In `external` stuff, the dependency tree gets truncated, and no information about which MPI was used will be propagated…